examples/ReduxExample/src/reducers/index.js

```diff
@@ -4,27 +4,31 @@ import { NavigationActions } from 'react-navigation';
 import { AppNavigator } from '../navigators/AppNavigator';
 
 // Start with two routes: The Main screen, with the Login screen on top.
-const initialNavState = {
-  index: 1,
-  routes: [
-    { key: 'InitA', routeName: 'Main' },
-    { key: 'InitB', routeName: 'Login' },
-  ],
-};
-
-const initialAuthState = { isLoggedIn: false };
+const firstAction = AppNavigator.router.getActionForPathAndParams('Main');
+const tempNavState = AppNavigator.router.getStateForAction(firstAction);
+const secondAction = AppNavigator.router.getActionForPathAndParams('Login');
+const initialNavState = AppNavigator.router.getStateForAction(secondAction, tempNavState);
```

Comment:
Unfortunately, I agree it's more complicated, but it does also prevent silly user errors. Given the number of people who couldn't figure out how to use it and got errors like in #485 it seems like this is a better approach. I'd love to write it a simpler way if possible -- any ideas?